packages/link/src/Link.js

```diff
@@ -33,7 +33,6 @@ const linkStyles = (props, baseTheme) => {
     Link_color_hover: baseTheme.color_link_hover,
     Link_color_focus: baseTheme.color_link_focus,
     Link_color_active: baseTheme.color_link_active,
-    Link_fontSize: `${2 * baseTheme.fontSize_base}px`,
```

Comment:
At the moment, this is the only component that must be updated along with `fontSize_base`. :(